src/shellstrings.c

```diff
@@ -7,88 +7,331 @@
+        printf(BOLDWHITE "c" RESET "                        : Enable/Disable color\n");
+        printf(BOLDWHITE "h" RESET "                        : Help\n");
+        printf(BOLDWHITE "q" RESET "                        : Quit program\n");
+    }
+
+    else {
+        printf("\nPlease load a file to begin. "
+               "Additionally, select an output mode and/or "
+               "choose dictionary before running the program.\n\n");
+
+        printf("f [/path/file.txt]       : Input text file\n");
+        printf("d [/path/dictionary.txt] : Input custom dictionary file\n");
+        printf("m [number]               : Mode [1 - Quiet, 2 - Verbose, 3 - Interactive]\n");
+        printf("c                        : Enable/Disable color\n");
+        printf("h                        : Help\n");
+        printf("q                        : Quit program\n");
+    }
 }
 
 /* See shellstrings.h */
-void shell_print(char** lines) {
+void shell_save(bool *color) {
+    if (*color == true) {
+        printf(BOLDWHITE "p" RESET " : Print all modifications\n");
+        printf(BOLDWHITE "s" RESET " : Save to existing file\n");
+        printf(BOLDWHITE "c" RESET " : Save to custom file\n");
+        printf(BOLDWHITE "r" RESET " : Re-edit file\n");
+        printf(BOLDWHITE "q" RESET " : Quit program\n");
+    }
+
+    else {
+        printf("p : Print all modifications\n");
+        printf("s : Save to existing file\n");
+        printf("c : Save to custom file\n");
+        printf("r : Re-edit file\n");
+        printf("q : Quit program\n");
+    }
+}
+
+/* See shellstrings.h */
+void shell_edit_success(bool *color) {
+    if (*color == true) {
+        printf(GREEN "\nSpellcheck complete.\n" RESET);
+    }
+
+    else {
+        printf("\nSpellcheck complete.\n");
+    }
+}
+
+/* See shellstrings.h */
+void shell_print(char **lines) {
     int i = 0;
 
-    printf("\n\n");
     while (lines[i] != NULL) {
         printf("%s", lines[i]);
+        
         i++;
     }
-    printf("\n\n");
 }
 
-
 /* See shellstrings.h */
-void shell_error(char* error_text) {
-	printf("=> ERROR: %s <=\n", error_text);
+void shell_input(char *filename, char *status, bool *color) {
+    if (*color == true) {
+        printf(BOLDWHITE "%s" RESET " has been accepted as " BOLDWHITE "%s" RESET ".\n", filename, status);
+    }
+
+    else {
+        printf("%s has been accepted as %s.\n", filename, status);
+    }
 }
 
 /* See shellstrings.h */
-void shell_usage() {
-    printf("Usage: ./spellcheck [-flag] [~/path/file.txt] [-saveflag] [~/path/destination.txt]\n");
+void shell_error(char *error_text, bool *color) {
+    if (*color == true) {
+        printf(RED "ERROR:" RESET " %s", error_text);
+    }
+
+    else {
+        printf("ERROR: %s", error_text);
+    }
 }
 
 /* See shellstrings.h */
-void shell_prompt() {
-    printf("\nspellcheck > ");
+void shell_usage(bool *color) {
```

Comment:
Why is this a `bool *` and not just a `bool`?

Reply:
changed!